TvMaterialCatalog/app/src/main/java/com/google/tv/material/catalog/HomeGridCard.kt

```diff
@@ -7,26 +7,29 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
+import androidx.tv.material3.Card
 import androidx.tv.material3.CardDefaults
-import androidx.tv.material3.CardLayoutDefaults
-import androidx.tv.material3.ExperimentalTvMaterial3Api
-import androidx.tv.material3.StandardCardLayout
+import androidx.tv.material3.StandardCardContainer
 import androidx.tv.material3.Text
 
-@OptIn(ExperimentalTvMaterial3Api::class)
 @Composable
 fun ComponentsGridCard(
     component: Component,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    onClick: () -> Any = {}
```

Comment:
he callback should not have a return value, as it is to deliver click events to the outer component.

If we need to handle the result of function calls that potentially fails, we should introduce another component to manage state as recommended in [this doc](https://developer.android.com/develop/ui/compose/state#state-hoisting).